src/post_section/post_section.jsx

```diff
@@ -30,102 +31,63 @@ function LinksList({ link }) {
         alt={link.name} /></Link>
     </td>;
 }
-class PostSection extends React.Component {
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        return (
-            <div>
-                <section className="second_section">
-                    <div className="second_section_block">
-                        <div className="author_of_post_info">
-                            <div className="inf_about_author">
-                                <img src="/photos/avatar.png" width="56" height="56" alt="avatar" />
-                                <div className="text_about_author">
-                                    <p className="author_name">Mika Matikainen</p>
-                                    <p className="time"> <time datetime="2020-04-15">Apr 15, 2020</time> · 4 min read</p>
-                                </div>
+const PostSection = () => {
+
+    const [data, setPosts] = useState({ posts: [] })
+    useEffect(() => {
+        fetch(`https://dolphin-app-cbjj4.ondigitalocean.app/posts/${index}`)
+            .then(response => response.json())
+            .then(data => setPosts({ posts: data }))
+    })
+    const formattedDate = new Date(data.posts.updated_at).toLocaleDateString('en-US', { year: 'numeric', month: 'short', day: 'numeric' });
+    const paramsString = document.location.href;
+    const index = paramsString.split("/").slice(-1)[0] 
```

Comment:
use [useParams](https://reactrouter.com/en/main/hooks/use-params) hook to get post id from url